decls/index.js

```diff
@@ -99,6 +107,10 @@ declare type Prettier$SyntaxError = {
   message: string,
   stack: string,
 };
+declare type Prettier$CursorResult = {
+  formatted: any,
+  cursorOffset: number,
+};
```

Comment:
if this is just for our own internal use inside of `executePrettierOnBufferRange`, we should declare inside of that file. These typedefs are more for things that are third-party dependencies and/or things that are used globally throughout the app.

Reply:
I mean it is the type definition for an external resource. I'm happy to declare in `executePrettierOnBufferRange ` if you still wish for that.

Reply:
Oh sorry, I thought this was just an internal thing you made up. Let's leave for now then 👍